index.html

```diff
@@ -134,6 +137,329 @@ <h1 class="banner-header banner-content">
+                data="assets/illustrations/ellipse-1.svg"
+              ></object>
+            </li>
+            <li><p class="overline-text-2">Back End Dev</p></li>
+            <li>
+              <object
+                class="overline-ellipse"
+                data="assets/illustrations/ellipse-1.svg"
+              ></object>
+            </li>
+            <li><p class="overline-text-2">2015</p></li>
+          </ul>
+        </div>
+
+        <p class="body-text-2">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+
+        <div class="work-card-techstack">
+          <ul>
+            <li><p class="sub-text-2">html</p></li>
+            <li><p class="sub-text-2">css</p></li>
+            <li><p class="sub-text-2">javaScript</p></li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="button" class="work-card-button button-text-1">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <!--ABOUT AREA-->
+    <section id="About" class="body-content-2">
+      <!--About Content-->
+      <div class="about-content">
+        <h1 class="banner-header">About Myself</h1>
+
+        <p class="body-text-2">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+
+        <p class="head-text-1">LET'S CONNECT</p>
+
+        <div class="banner-socials">
+          <ul class="banner-socials">
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/twitter.svg"
+                  alt="Twitter"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/linkedin-in.svg"
+                  alt="Linkedin"
+              /></a>
+            </li>
+
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/medium.svg"
+                  alt="Linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/github.svg"
+                  alt="Github"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="banner-social-icon"
+                  src="assets/icons/discord.svg"
+                  alt="Github"
+              /></a>
+            </li>
+          </ul>
+        </div>
+
+        <div>
+          <button type="button" class="work-card-button button-text-1">See Project</button>
```

Comment:
The comment for the button above applies here as well. Additionally, the text in the template is `Get my resume`, not `See Project`. Kindly fix this.